### PR TITLE
[#786] Errors for input fields

### LINKED ISSUE #786: Errors for input fields
On a send screen there are two input fields. One for address and one for the amount. Bring back those error labels notifying users about
- insufficient funds
- invalid address

### PR DESCRIPTION
**Pull request by LukasKorba**
Closes #786 

- Send flow updated so both input field show an error

Simulator Screenshot - iPhone 14 Pro - 2023-07-19 at 09 20 15

This code review checklist is intended to serve as a starting point for the author and reviewer, although it may not be appropriate for all types of changes (e.g. fixing a spelling typo in documentation).  For more in-depth discussion of how we think about code review, please see [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md).

# Author

- [x] Self-review: Did you review your own code in GitHub's web interface? Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs.
- [ ] Does the code abide by the [Coding Guidelines](../blob/main/docs/CODING_GUIDELINES.md)?
- [ ] Automated tests: Did you add appropriate automated tests for any code changes?
- [ ] Code coverage: Did you check the code coverage report for the automated tests?  While we are not looking for perfect coverage, the tool can point out potential cases that have been missed.
- [ ] Documentation: Did you update Docs as appropiate? (E.g [README.md](../blob/main/README.md), etc.)
- [x] Run the app: Did you run the app and try the changes? 
- [ ] Did you provide Screenshots of what the App looks like before and after your changes as part of the description of this PR? (only applicable to UI Changes)
- [x] Rebase and squash: Did you pull in the latest changes from the main branch and squash your commits before assigning a reviewer? Having your code up to date and squashed will make it easier for others to review. Use best judgement when squashing commits, as some changes (such as refactoring) might be easier to review as a separate commit.

# Reviewer

- [x] Checklist review: Did you go through the code with the [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md) checklist?
- [x] Ad hoc review: Did you perform an ad hoc review?  _In addition to a first pass using the code review guidelines, do a second pass using your best judgement and experience which may identify additional questions or comments. Research shows that code review is most effective when done in multiple passes, where reviewers look for different things through each pass._
- [ ] Automated tests: Did you review the automated tests?
- [ ] Manual tests: Did you review the manual tests?_You will find manual testing guidelines under our [manual testing section](../blob/main/docs/testing/manual_testing)_
- [ ] How is Code Coverage affected by this PR? _We encourage you to compare coverage before and after your changes and when possible, leave it in a better place. [Learn More...](../blob/main/docs/testing/local_coverage.md)_
- [ ] Documentation: Did you review Docs, [README.md](../blob/main/README.md), [LICENSE.md](../blob/main/LICENSE.md), and [Architecture.md](../blob/main/docs/Architecture.md) as appropriate?
- [ ] Run the app: Did you run the app and try the changes? While the CI server runs the app to look for build failures or crashes, humans running the app are more likely to notice unexpected log messages, UI inconsistencies, or bad output data.

### CODE REVIEW

File: modules/Sources/Generated/Resources/Localizable.strings [hunk 1]
```diff
@@ -118,6 +118,9 @@
 "send.memo" = " memo: %@";
 "send.includeMemo" = "Want to include memo? Tap here.";
 "send.editMemo" = "Memo included. Tap to edit.";
+"send.error.insufficientFunds" = "insufficient funds";
+"send.error.invalidAmount" = "invalid amount";
+"send.error.invalidAddress" = "invalid address";
```
Comment (HonzaR) [inline modules/Sources/Generated/Resources/Localizable.strings:123]: Could these texts start with a big letter? Our Material design guidelines recommend that for the Android side. 
See [documentation](https://m3.material.io/components/text-fields/specs#f157ea73-f2fd-4d99-aab3-c96bd9c91eec ).

File: modules/Sources/Generated/L10n.swift [hunk 2]
```diff
@@ -484,6 +484,14 @@ public enum L10n {
     public static let succeeded = L10n.tr("Localizable", "send.succeeded", fallback: "Sending transaction succeeded")
     /// Send Zcash
     public static let title = L10n.tr("Localizable", "send.title", fallback: "Send Zcash")
+    public enum Error {
+      /// insufficient funds
+      public static let insufficientFunds = L10n.tr("Localizable", "send.error.insufficientFunds", fallback: "insufficient funds")
+      /// invalid address
+      public static let invalidAddress = L10n.tr("Localizable", "send.error.invalidAddress", fallback: "invalid address")
+      /// invalid amount
+      public static let invalidAmount = L10n.tr("Localizable", "send.error.invalidAmount", fallback: "invalid amount")
```
Comment (HonzaR) [inline modules/Sources/Generated/L10n.swift:493]: Our [issue](https://github.com/zcash/secant-android-wallet/issues/340) also mentions a few more errors we could handle. I didn't analyze how much we can align them with what the SDK provides, but I rather put the question here.

**Review by HonzaR — APPROVED** [review]
I left two comments inline. Other than that, it seems good to me.

File: modules/Sources/Generated/Resources/Localizable.strings (hunk 1 above)
Comment (LukasKorba) [reply modules/Sources/Generated/Resources/Localizable.strings:123]: We don't have such guidelines so normally I would leave it to designers/copywriters/anyone but me. I updated it anyway.

File: modules/Sources/Generated/L10n.swift (hunk 2 above)
Comment (LukasKorba) [reply modules/Sources/Generated/L10n.swift:493]: I think the `fee` feedback looks interesting. Other 4 are probably covered on our side. If the fee is agreed to do, we will doit in a follow up ticket.

**Pull request merged**